bazel/foreign_cc/luajit.patch

```diff
@@ -125,3 +125,65 @@ index 00000000..3b4f8eca
 +"${MAKE:-make}" -j$(nproc) V=1 PREFIX="$PREFIX" \
 +  "${EXTRA_MAKE_ARGS[@]}" \
 +  install
+diff --git a/luajit_build_win.sh b/luajit_build_win.sh
+new file mode 100755
+index 00000000..8fb0fbe8
+--- /dev/null
++++ b/luajit_build_win.sh
+@@ -0,0 +1,39 @@
++#!/bin/bash
++
++set -e
++
++PREFIX=""
++while [[ $# -gt 0 ]]; do
++    case $1 in
++        --prefix=*)
++            PREFIX="${1#*=}"
++            shift
++            ;;
++        --prefix)
++            PREFIX="$2"
++            shift 2
++            ;;
++        *)
++            shift
++            ;;
++    esac
++done
++
++# Copy source tree to a build directory
++SRC_DIR="$(dirname "$(realpath "$0")")"
++BUILD_DIR="$(basename "$SRC_DIR")"
++cp -r "$SRC_DIR" "$BUILD_DIR"
++cd "$BUILD_DIR"
++
++ ls
++ cd src
++./msvcbuild.bat static
```

Comment:
in this cases its obviously harder to upstream the patch - but im wondering if this .bat stuff is really needed and if it is whether it can be incorporated to existing patch/script - eg by setting an env var and calling it here if set

Reply:
I was able to remove the .bat patch, instead making sure to copy `vc140.pdb` to the output prefix, which is required when using `/Zi` as msvcbuild.bat forces to use.

I still kept the separate build_win.sh since it seems worth keeping it independent from the unix one